flutter_auth_ui/ios/Classes/SwiftFlutterAuthUiPlugin.swift

```diff
@@ -17,6 +17,20 @@ public class SwiftFlutterAuthUiPlugin: NSObject, FlutterPlugin, FUIAuthDelegate
         registrar.addApplicationDelegate(instance)
     }
 
+//  #############################################################
```

Comment:
I'm sorry, but I want you to match the style of comments with others. 
I don't think it's good to separate with # because it's not Swift's document style.

Reply:
Fixed